sbomgrader/translate/translation_map.py

```diff
@@ -232,26 +314,38 @@ def convert(
             )
             for fallback in doc.sbom_format_fallback
         ), f"This map cannot convert from {doc.sbom_format}."
+        # Preprocess
+        dict_ = doc.doc
```

Comment:
I would like to see a better variable names across the codebase. The variable `dict_` doesn't say anything to a reviewer. I had to re-read this section several times to understand what is happening here. If the variable would have meaningfull name it would simplifies the code reading.

Reply:
Sorry about that :sweat_smile: Updated.